includes/class-edd-html-elements.php

```diff
@@ -504,7 +504,7 @@ public function select( $args = array() ) {
 		$class  = implode( ' ', array_map( 'sanitize_html_class', explode( ' ', $args['class'] ) ) );
 		$output = '<select' . $disabled . $readonly . ' name="' . esc_attr( $args['name'] ) . '" id="' . esc_attr( edd_sanitize_key( str_replace( '-', '_', $args['id'] ) ) ) . '" class="edd-select ' . $class . '"' . $multiple . ' data-placeholder="' . $placeholder . '"'. $data_elements . '>';
 
-		if ( ! isset( $args['selected'] ) || empty( $args['selected'] ) || ! $args['selected'] ) {
+		if ( ! isset( $args['selected'] ) || is_null( $args['selected'] ) || strlen( $args['selected'] ) === 0 || ! $args['selected'] ) {
```

Comment:
I think it's best that we do a check of:
`( is_array( $args['selected'] ) && empty( $args['selected'] )`

This way we're doing the intended of what we were doing before (since default is empty array), but still allowing a value of `0`.